lib/autoupdate/agent/updater.go

```diff
@@ -184,6 +186,10 @@ func NewLocalUpdater(cfg LocalUpdaterConfig, ns *Namespace) (*Updater, error) {
 				args = append(args, "--reload")
 			}
 			cmd := exec.CommandContext(ctx, name, args...)
+			cmd.Env = slices.Clone(os.Environ())
```

Comment:
Bug introduced here: this gets overridden below. Maybe delete this line and move 190 to after 198?

Reply:
Good catch, fixed